configuration/security.md

```diff
@@ -75,21 +75,20 @@ sudo deluser -remove-home pi
 ```
 This command will result in a warning that the group `pi` has no more members. The `deluser` command removes both the `pi` user and the `pi` group though, so the warning can be safely ignored.
 
+To change autologin of the 'pi' user to the 'alice' user, use:
+
+```bash
+sudo sed -i '/autologin-user=pi/autologin-user=alice/g' /etc/lightdm/lightdm.conf
+```
 
 ## Make `sudo` require a password
 
 Placing `sudo` in front of a command runs it as a superuser, and by default, that does not need a password. In general, this is not a problem. However, if your Pi is exposed to the internet and somehow becomes exploited (perhaps via a webpage exploit for example), the attacker will be able to change things that require superuser credentials, unless you have set `sudo` to require a password.
 
-To force `sudo` to require a password, enter:
-
-```bash
-sudo visudo /etc/sudoers.d/010_pi-nopasswd
-```
-
-and change the `pi` entry (or whichever usernames have superuser rights) to:
+To force `sudo` to require a password, use:
 
 ```bash
-pi ALL=(ALL) PASSWD: ALL
+sudo sed -i '/pi/alice/g' /etc/sudoers.d/010_pi-nopasswd
```

Comment:
IIRC it's always recommended to use the `visudo` command for editing the sudoers file, as it guards against accidental mistakes?

Reply:
I think you're able to change it back to what it was, but there's no mistake.